provisioner/data.js

```diff
@@ -265,83 +387,51 @@ WorkerType.createLaunchSpec = function (region, instanceType, worker, keyPrefix,
   var regionSpecificKeys = [
```

Comment:
Should be made `const` I see no reason not to make it a global `const REGION_SPECIFIC_KEYS = [...];` list.